pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,62 @@
+{stdenv, python, gettext, qt5, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  build_python = python.withPackages (ps: [ps.setuptools]);
+
+  nativeBuildInputs = [ 
+    gettext 
+    build_python
+    qt5.wrapQtAppsHook
+  ];
+
+  app_python = python.withPackages (ps: with ps; [
+      pyqt5
+      send2trash
+      sphinx
+      polib
+      hsaudiotag3k
+  ]);
+
+  buildInputs = [
+    app_python
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make i18n
+    make modules
+    make qt/dg_rc.py
+    make pyc
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make PREFIX=$out install --assume-old=all --assume-old=pyc
```

Comment:
```suggestion
    installFlags = [
      "PREFIX=${placeholder ''out''}"
      "--assume-old=all"
      "--assume-old=pyc"
    ];
```

Reply:
@jtobin what about `chmod +x $out/share/dupeguru/run.py` as I guess the python file won't get executable.

Reply:
Is that really necessary? I would not expect scripts in `$out/share` to be executed directly. But if it is necessary, you can add it to `postInstall` and add a comment explaining how is it getting executed.